frontend/src/components/table/schedule/schedule-parts/shift-cell.component.tsx

```diff
@@ -1,41 +1,24 @@
-import React, { useState } from "react";
-import { ColorProvider } from "../../../../helpers/colors/color.provider";
-import { VerboseDate } from "../../../../logic/real-schedule-logic/month.logic";
+import React from "react";
 import { ShiftCode } from "../../../../state/models/schedule-data/shift-info.model";
 import { BaseCellComponent } from "./base-cell.component";
 import { CellOptions } from "./cell-options.model";
 
-function getShiftCode(value: string): ShiftCode {
-  return ShiftCode[value] || ShiftCode.W;
+function getShiftCode(value: string | number): ShiftCode {
+  return typeof value === "number" ? value.toString() : ShiftCode[value] || ShiftCode.W;
```

Comment:
To potrzeba będzie usunąć. Jest związane z tym, że używamy ten typ komórki przy pokazywaniu nadgodzin